### PR TITLE
packs: adding checks for known bad python packages

### PR DESCRIPTION
**Pull request by muffins**
Adds a query to search for the backdoored Python packages detailed here: http://www.nbu.gov.sk/skcsirt-sa-20170909-pypi/index.html

### CODE REVIEW

**Comment by muffins** [comment]
Just as a heads up on this, this query pack is looking for the list of masqueraded package names detailed below:
```
List of fake package names:
– acqusition (uploaded 2017-06-03 01:58:01, impersonates acquisition)
– apidev-coop (uploaded 2017-06-03 05:16:08, impersonates apidev-coop_cms)
– bzip (uploaded 2017-06-04 07:08:05, impersonates bz2file)
– crypt (uploaded 2017-06-03 08:03:14, impersonates crypto)
– django-server (uploaded 2017-06-02 08:22:23, impersonates django-server-guardian-api)
– pwd (uploaded 2017-06-02 13:12:33, impersonates pwdhash)
– setup-tools (uploaded 2017-06-02 08:54:44, impersonates setuptools)
– telnet (uploaded 2017-06-02 15:35:05, impersonates telnetsrvlib)
– urlib3 (uploaded 2017-06-02 07:09:29, impersonates urllib3)
– urllib (uploaded 2017-06-02 07:03:37, impersonates urllib3)
```
That doesn't mean to say that these names will always be bad.

File: packs/vuln-management.conf [hunk 1]
```diff
@@ -132,6 +132,14 @@
       "version" : "1.4.5",
       "description" : "Retrieves all application bundles using unauthenticated Sparkle update feeds. See (https://vulnsec.com/2016/osx-apps-vulnerabilities/) for details.",
       "value" : "Tracking vulnerable applications updates may allow blocking of DNS or removal by BundleID."
+    },
+    "backdoored_python_packages": {
+      "query" : "select name as package_name, version as package_version, path as package_path from python_packages where package_name = 'acqusition' OR package_name = 'apidev-coop' OR package_name = 'bzip' OR package_name = 'crypt' OR package_name = 'django-server' OR package_name = 'pwd' OR package_name = 'setup-tools' OR package_name = 'telnet' OR package_name = 'urlib3' OR package_name = 'urllib';",
+      "interval" : "86400",
+      "platform" : "darwin,linux",
```
Comment (mtmcgrew) [inline packs/vuln-management.conf]: nit, can you do 'posix'

**Comment by osqueryer** [comment]
:-1: The commit d5f348001e7cf97a1559ca6ecc88ce2b32b88ff2 (Job results: [5808](https://jenkins.osquery.io/job/osqueryPullRequestLinux/5808/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit d5f348001e7cf97a1559ca6ecc88ce2b32b88ff2 (Job results: [5809](https://jenkins.osquery.io/job/osqueryPullRequestLinux/5809/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit d5f348001e7cf97a1559ca6ecc88ce2b32b88ff2 (Job results: [2691](https://jenkins.osquery.io/job/osqueryPullRequestWindows/2691/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit d5f348001e7cf97a1559ca6ecc88ce2b32b88ff2 (Job results: [2692](https://jenkins.osquery.io/job/osqueryPullRequestWindows/2692/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit d5f348001e7cf97a1559ca6ecc88ce2b32b88ff2 (Job results: [735](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/735/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit d5f348001e7cf97a1559ca6ecc88ce2b32b88ff2 (Job results: [736](https://jenkins.osquery.io/job/osqueryPullRequestFreeBSD/736/)) failed one or more tests (**FreeBSD**).

**Pull request merged**